.env.template

```diff
@@ -1,14 +0,0 @@
-REST_PORT=3000
```

Comment:
Why did you delete the .env.template file? This must never be deleted - if anyone else now needs to work on this code, it will be very difficult what to configure.  When devops needs to give this service all the config in jenkins, it will be almost impossible for them. Please add this file back - but ensure in this file that there are no sensitive information.

Reply:
Ouch. my bad! must have forgotten to revert. Thanks